### DISCUSSION TITLE
Styling Problem

### DISCUSSION THREAD

**Discussion by raiyansarker** [vercel/next.js#23119] [Help]
I just encountered this problem where the global CSS is not loading on the production build of Next JS project. I am using tailwind CSS along with scss, I imported everything of tailwind in global.scss and then I imported that in _app.js. And also I created _document.js file. The file looks like this, 
`import Document, { Html, Head, Main, NextScript } from "next/document";

class MyDocument extends Document {
  static async getInitialProps(ctx) {
    const initialProps = await Document.getInitialProps(ctx);
    return { ...initialProps };
  }

  render() {
    return (
      <Html lang="en">
        <Head />
        <body>
          <Main />
          <NextScript />
        </body>
      </Html>
    );
  }
}

export default MyDocument;
`

But when I deployed the app in Vercel, I found that I am not getting global css. So I didn't edited anything, I just deleted _document.js and redeployed and now the style is working. So clearly the problem was in _document.js but where? I mean what I didn't imported in _document.js which may cause the problem.

Thanks in advance!

**Comment by raiyansarker** [comment]
The code didn't load there, here is the code for _document.js
```
import Document, { Html, Head, Main, NextScript } from "next/document";

class MyDocument extends Document {
  static async getInitialProps(ctx) {
    const initialProps = await Document.getInitialProps(ctx);
    return { ...initialProps };
  }

  render() {
    return (
      <Html lang="en">
        <Head />
        <body>
          <Main />
          <NextScript />
        </body>
      </Html>
    );
  }
}

export default MyDocument;
```

**Comment by leerob** [comment]
> I am using tailwind CSS along with scss, I imported everything of tailwind in global.scss

Why not just use Tailwind? Regardless, have you started from the Tailwind example?
